README.md

```diff
@@ -6,6 +6,7 @@ This project is still very much a Work In Progress.
 
 ## Requirements
 
+* AWS
```

Comment:
bit pedantic but this should be aws-cli

Reply:
No, it was AWS :). You need AWS to run this.

Reply:
Ah, fair enough